<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportSMP</class>
 <widget class="QDialog" name="ImportSMP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>Импортировать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="loLoadForm">
     <item>
      <widget class="QLabel" name="lblLoadForm">
       <property name="text">
        <string>Загрузить из</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFileName"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnSelectFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTextBrowser" name="logBrowser"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Тип поля ID_DOC</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cmbMod">
     <item>
      <property name="text">
       <string>ID врача</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Код врача</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Региональный код врача</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Федеральный код врача</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Врач по умолчанию</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ImportSMP</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
